Update proposal research question (#1)
UBC-MDS/movielover_py: docs/proposal.md

TBC

## Research Questions and Usage Scenarios
### Research questions:

1. How does the US gross revenue different by movie genre?

2. What is the relationship between rating and duration of the movie?

> 3. What are the top 10 rated movies at the year(s) of interest? 

Reviewer: ```suggestion
3. What are the top 10 rated movies at the year(s) of interest by genre? 
```

Reviewer 2: > What are the top 10 rated movies at the year(s) of interest (by genre)?

> 3. What are the top 10 rated movies at the year(s) of interest by genre? 

Reviewer: ```suggestion
3. What are the top 10 rated movies at the year(s) of interest (by genre)? 
```
As per discussed in the Slack with Junrong, we think it's better to include () as genre is optional

### Scenario of usage:

> Kate is a movie enthusiast in the U.S. and she is interested in a broad range of topics and fun facts about the movie industry in North America at different years since 1990. She has a habit to watch a popular movie every week to understand why the movie was so popular in the specific timeframe. She thinks that the best way to understand is to watch it and give her own verdict. Kate also likes doing study how the movie industry is growing and changing by genre popularity and movie duration. She would like to use the gross revenue of movie as an indicator of box office and compare them across genres. As she observed more movies are extending their runtime, she would also like to find out if runtime of a movie is correlated with rating. She could possibly discovers a new trend that the highest rated movie might not be in the same genre that have highest gross revenue (highest box office).

Reviewer: ```suggestion
Kate is a movie enthusiast in the U.S. and she is interested in a broad range of topics and fun facts about the movie industry in North America at different years since 1990. She has a habit to watch a popular movie every week to understand why the movie was so popular in a specific timeframe. She thinks that the best way to understand is to watch it and give her own verdict. Kate also likes studying how the movie industry is growing and changing by genre popularity and movie duration. She would like to use the gross revenue of movies as an indicator of box office and compare them across genres. As she observed more movies are extending their runtime, she would also like to find out if the runtime of a movie is correlated with rating. She could possibly discover a new trend that the highest-rated movie might not be in the same genre that has the highest gross revenue (highest box office).
```
